### ISSUE TITLE
Kaiju won't run

### ISSUE THREAD

**Issue by Thomieh73** [sunbeam-labs/sbx_kaiju#5]
Hi Guys,

I have set-up the kaiju extension as described on the sbx_kaiju readme.

Now I am trying to run sunbeam and it finds the extension, but it is not starting any jobs with the extension.
here is the output of the run:

``` 
Running: snakemake --snakefile /work/projects/nn9305k/src/sunbeam-stable/Snakefile --configfile /work/projects/nn9305k/air_sample/analysis/pilot_exp/sunbeam_config.yml --cluster-config /work/projects/nn9305k/air_sample/configuration/cluster.json --cluster sbatch --account={cluster.account} --partition={cluster.partition} --mem-per-cpu={cluster.memcpu} --cpus-per-task={threads} --time={cluster.time} --job-name={cluster.name} -j 100 -w 90
Found extension sbx_kaiju in folder sbx_kaiju
Collecting host/contaminant genomes... done.
Collecting target genomes... done.
Building DAG of jobs...
Nothing to be done.
Complete log: /work/projects/nn9305k/air_sample/analysis/pilot_exp/.snakemake/log/2019-09-19T132112.143945.snakemake.log
``` 

my sunbeam_config file looks like this:

```
# 
# Sunbeam configuration file
#
# Defining samples:
#   Run 'sunbeam list_samples <data_dir>' to create a list of samples and
#   associated fastq.gz files. Samples must be in gzipped fastq format.
#
# Paths:
#   Paths are resolved through the following rules:
#     1. If the path is absolute, the path is parsed as-is
#     2. If the path is not absolute, the path at 'root' is appended to it
#     3. If the path is not 'output_fp', the path is checked to ensure it exists
#
# Suffixes:
#   Each subsection contains a 'suffix' key that defines the folder under
#   'output_fp' where the results of that section are put.
#

# General options
all:
  root: /work/projects/nn9305k/air_sample/analysis/pilot_exp
  output_fp: sunbeam_output
  samplelist_fp: samples.csv
  paired_end: true
  download_reads: false
  version: 2.0.3+dev0.gb2140b8.d20190830

# Quality control
qc:
  suffix: qc
  # Trimmomatic
  threads: 4
  java_heapsize: 512M
  leading: 3
  trailing: 3
  slidingwindow: [4, 15]
  minlen: 36
  adapter_fp: /usit/abel/u1/thhaverk/tools/anaconda3/envs/sunbeam/share/trimmomatic/adapters/NexteraPE-PE.fa
  # Cutadapt
  fwd_adapters: [GTTTCCCAGTCACGATC, GTTTCCCAGTCACGATCNNNNNNNNNGTTTCCCAGTCACGATC]
  rev_adapters: [GTTTCCCAGTCACGATC, GTTTCCCAGTCACGATCNNNNNNNNNGTTTCCCAGTCACGATC]
  # Komplexity
  kz_threshold: 0.55
  # Decontam.py
  pct_id: 0.5
  frac: 0.6
  host_fp: '/work/projects/nn9305k/db_flatfiles/contamination_hosts'


# Taxonomic classifications
classify:
  suffix: classify
  threads: 16
  kraken_db_fp: '/work/projects/nn9305k/db_flatfiles/kraken_databases/kraken_db_56G'

# Contig assembly
assembly:
  suffix: assembly
  min_length: 300
  threads: 16

# Contig annotation
annotation:
  suffix: annotation
  min_contig_len: 500
  circular_kmin: 10
  circular_kmax: 1000
  circular_min_len: 3500

blast:
  threads: 8

blastdbs:
  root_fp: ''

mapping:
  suffix: mapping
  genomes_fp: '/work/projects/nn9305k/genome_references/genomes/campylobacter'
  samtools_opts: '-F4'
  threads: 16

download:
  suffix: download
  threads: 4
  
sbx_kaiju:
  suffix: kaiju
  nodes_fp: '/work/projects/nn9305k/db_flatfiles/kaiju_database/nodes.dmp'
  names_fp: '/work/projects/nn9305k/db_flatfiles/kaiju_database/names.dmp'
  db_fp: '/work/projects/nn9305k/db_flatfiles/kaiju_database/nr_euk/kaiju_db_nr_euk.fmi'
  threads: 6
```

So now I am wondering why this extension is not starting. Where could I find the hint for that?

The rules for the kaiju extension are in the original source folder:
`/sunbeam_stable/extensions/sbx_kaiju.` 

But it is not in the folder:  `sunbeam-stable/sunbeam/extensions`

**Comment by Thomieh73** [comment]
I figured out what I missed to get this working,

In the usage instructions of this extension it does not indicate how to specify the `all_kaiju` rule.

That left me searching for how to do that and I found the answer in the report extension documentation.

I need to use it like this:

    sunbeam run --configfile ~/path/to/sunbeam_config.yml --use-conda all_kaiju

And then I see that kaiju is added to my list of jobs.

I will add that the documentation of this extension.
